doc/symbols.rst

```diff
@@ -62,16 +62,11 @@ for ES2015 symbols (and reserved for other future uses) in Duktape 2.x.
 +-----------------------------------------------+-----------------------------------------------------------------+
 | Internal string format                        | Description                                                     |
 +-----------------------------------------------+-----------------------------------------------------------------+
-| <ff> SomeUpperCaseValue                       | Hidden symbol (Duktape specific) used by Duktape internals.     |
-|                                               | Previously called internal properties.  First byte is 0xFF,     |
-|                                               | second is from [A-Z].                                           |
-+-----------------------------------------------+-----------------------------------------------------------------+
-| <ff> anyOtherValue                            | Hidden symbol (Duktape specific) used by application code.      |
-|                                               | First byte is 0xFF, second is ASCII (0x00-0x7f) but not         |
-|                                               | from [A-Z].                                                     |
-+-----------------------------------------------+-----------------------------------------------------------------+
-| <ff> <ff> anyOtherValue                       | Hidden symbol (Duktape specific) used by application code.      |
-|                                               | First and second bytes are 0xFF, remaining bytes arbitrary.     |
+| <ff> anyValue                                 | Hidden symbol (Duktape specific) used by application code.      |
+|                                               | Prior to Duktape 2.2 Duktape internal hidden symbols also used  |
```

Comment:
It might be nice to add a blank line above this. Easier to ignore that way.

Reply:
I'm not sure what you mean - do you mean adding a blank line between the first sentence and the rest of the Description? If so, that would make the entry inconsistent with the other description fields which don't separate a leading sentence from further explanation.

Reply:
alrighty, sounds good.